src/migrations/1.0.0-beta.31.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use chronicle::{
+    db::{
+        collections::OutputCollection, mongodb::config as mongocfg, MongoDb, MongoDbCollection, MongoDbCollectionExt,
+        MongoDbConfig,
+    },
+    types::stardust::block::output::{AliasId, NftId, OutputId},
+};
+use clap::Parser;
+use futures::TryStreamExt;
+use mongodb::{bson::doc, options::IndexOptions, IndexModel};
+use serde::Deserialize;
+
+#[derive(Parser, Debug)]
+#[command(author, version, about)]
+pub struct MongoDbArgs {
+    /// The MongoDb connection string.
+    #[arg(
+        long,
+        value_name = "CONN_STR",
+        env = "MONGODB_CONN_STR",
+        default_value = mongocfg::DEFAULT_CONN_STR,
+    )]
+    pub mongodb_conn_str: String,
+    /// The MongoDb database name.
+    #[arg(long, value_name = "NAME", default_value = mongocfg::DEFAULT_DATABASE_NAME)]
+    pub mongodb_database_name: String,
+}
+
+#[tokio::main]
+async fn main() -> eyre::Result<()> {
+    let args = MongoDbArgs::parse();
+    let config = MongoDbConfig {
+        conn_str: args.mongodb_conn_str,
+        database_name: args.mongodb_database_name,
+    };
```

Comment:
What's nice about subcommands is that they can reuse the already existing MongoDb args struct, so we could get rid of this code.